include/nsysccr/cdc.h

```diff
@@ -92,6 +93,15 @@ struct WUT_PACKED CCRCDCEepromData
 WUT_CHECK_OFFSET(CCRCDCEepromData, 0x0, version);
 WUT_CHECK_SIZE(CCRCDCEepromData, 0x304);
 
+struct WUT_PACKED CCRCDCWowlWakeDrcArg
+{
+   WUT_PADDING_BYTES(0x6);
+   //! Must be 1 or 2
+   uint8_t arg;
```

Comment:
What exactly is ´arg` used for? Is this the DRC destination to power on? If so it should be something like:
```suggestion
   CCRCDCDestination destination;
```

Reply:
```suggestion
   // 1 = Power on normally.
   // 2 = Connect in \c CCR_CDC_DRC_STATE_BACKGROUND state.
   uint8_t arg;
```
or
```suggestion
   //! Set to 1 to power on normally or 2 to connect in \c CCR_CDC_DRC_STATE_BACKGROUND state.
   uint8_t arg;
```

Maybe rename `arg` to something like `mode` or `state`?

Unless you want to use an enum:?
```
typedef enum CCRCDCWakeStateEnum
{
   CCR_CDC_WAKE_STATE_ACTIVE     = 1,
   CCR_CDC_WAKE_STATE_BACKGROUND = 2,
} CCRCDCWakeStateEnum;
```

Reply:
An enum with the comments above added to the enum sounds good to me!